checkmk_extensions/agent/scheduler_runner.ps1

```diff
@@ -0,0 +1,36 @@
+# Enable StrictMode 3.0
+Set-StrictMode -Version 3.0
+
+class Config {
+    [bool]$rcc
+
+    static [string] GetConfigContent($configFilePath) {
+        try {
+            $configFileContent = Get-Content -Raw -Path $configFilePath -ErrorAction Stop
+            return $configFileContent
+        }
+        catch {
+            throw
+        }
+    }
+
+    # Method to parse the config file (JSON) and set the value of rcc
+    static [Config] ParseConfigFile($configFilePath) {
+        $config = [Config]::new()
+
+        try {
+            $configFileContent = [Config]::GetConfigContent($configFilePath)
+            if ($null -ne $configFileContent -and $configFileContent -ne '') {
```

Comment:
Can we even encounter `$null` here? Also, I don't think we have to check for empty strings ourselves. Json will tell us if there is a problem.

Reply:
We could encounted `$null` if, for example the json file is present, but empty. The solution would be the same as above: to make the `rcc` parameter have a default value via the bakery rule.

You are right, we don't need to check for the empty string.